llvm/include/llvm/Transforms/Utils/ValueMapper.h

```diff
@@ -105,6 +105,13 @@ enum RemapFlags {
   /// Any global values not in value map are mapped to null instead of mapping
   /// to self.  Illegal if RF_IgnoreMissingLocals is also set.
   RF_NullMapMissingGlobalValues = 8,
+
+  /// Do not remap atom instances. Only safe if to do this if the cloned
+  /// instructions being remapped are inserted into a new function, or an
+  /// existing function where the inlined-at fields are updated. If in doubt,
+  /// don't use this flag. It's used for compiler performance reasons rather
+  /// than correctness.
```

Comment:
```suggestion
  /// don't use this flag. It's used when remapping is known to be un-necessary
  /// to save some compile-time.
```

Reply:
IMO suggesting that it's not related to correctness is misleading, because the presence/absence of the flag can lead to correctness issues. Better to just avoid saying that and say something else as suggested.